SayLess/__init__.py

```diff
@@ -18,10 +20,20 @@
     SECRET_KEY='CSE'
 )
 
-params = urllib.parse.quote_plus(get_secret("DB"))
+#SET UP TO USE FLASK MAIL
+app.config['MAIL_SERVER'] = 'smtp.gmail.com'
+app.config['MAIL_PORT'] = 465
+app.config['MAIL_USE_SSL'] = True
+app.config['MAIL_USE_TLS'] = False
+app.config['MAIL_USERNAME'] = 'sayless442@gmail.com'
+app.config['MAIL_PASSWORD'] = '4En1QVD4#M#J'
+mail = Mail(app)
+
+#params = urllib.parse.quote_plus(get_secret("DB"))
 
 app.config.from_pyfile('config.py', silent=True)
-app.config['SQLALCHEMY_DATABASE_URI'] = "mssql+pyodbc:///?odbc_connect={}".format(params)
+app.config['SQLALCHEMY_DATABASE_URI'] = 'mysql://rileybur:50216039@tethys.cse.buffalo.edu:3306/cse442_542_2020_fall_teamb_db'
```

Comment:
Can you change this please. Revert these changes to what's originally supposed to be here.